GooglePay/src/test/java/com/braintreepayments/api/googlepay/GooglePayCardNonceUnitTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.braintreepayments.api.googlepay
+
+import android.os.Parcel
+import com.braintreepayments.api.core.PostalAddress
+import com.braintreepayments.api.sharedutils.Json
+import com.braintreepayments.api.testutils.Assertions.assertBinDataEqual
+import com.braintreepayments.api.testutils.Fixtures
+import kotlinx.parcelize.parcelableCreator
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@RunWith(RobolectricTestRunner::class)
+class GooglePayCardNonceUnitTest {
+
+    private fun getPostalAddressObjectFromJson(address: JSONObject): PostalAddress {
+        val result = PostalAddress().apply {
+            recipientName = Json.optString(address, "name", "")
+            streetAddress = Json.optString(address, "address1", "")
+            extendedAddress = listOf(
+                Json.optString(address, "address2", ""),
+                Json.optString(address, "address3", "")
+            ).joinToString(separator = "\n").trim()
+            locality = Json.optString(address, "locality", "")
+            region = Json.optString(address, "administrativeArea", "")
+            countryCodeAlpha2 = Json.optString(address, "countryCode", "")
+            postalCode = Json.optString(address, "postalCode", "")
+        }
```

Comment:
Since `PostalAddress` is a Kotlin class that has it's properties defined in the constructor, we can invoke the constructor and pass in all of the params instead of having to create the `PostalAddress` instance then call the setter functions for each property.

```suggestion
        val result = PostalAddress(
            recipientName = Json.optString(address, "name", "")
            streetAddress = Json.optString(address, "address1", "")
            extendedAddress = listOf(
                Json.optString(address, "address2", ""),
                Json.optString(address, "address3", "")
            ).joinToString(separator = "\n").trim()
            locality = Json.optString(address, "locality", "")
            region = Json.optString(address, "administrativeArea", "")
            countryCodeAlpha2 = Json.optString(address, "countryCode", "")
            postalCode = Json.optString(address, "postalCode", "")
        )
```